examples/collector-exporter-node/start.js

```diff
@@ -4,6 +4,7 @@ const opentelemetry = require('@opentelemetry/api');
 // const { ConsoleLogger,  LogLevel} = require('@opentelemetry/core');
 const { BasicTracerProvider, ConsoleSpanExporter, SimpleSpanProcessor } = require('@opentelemetry/tracing');
 const { CollectorTraceExporter, CollectorMetricExporter, CollectorProtocolNode } = require('@opentelemetry/exporter-collector');
+const { MeterProvider } = require('@opentelemetry/metrics');
```

Comment:
I would rather have separate file for metrics and for tracer then trying to add more into this example